mindsdb/integrations/handlers/ms_teams_handler/ms_teams_handler.py

```diff
@@ -144,17 +144,39 @@ def get_chat_config(self) -> Dict:
         params = {
             'polling': {
                 'type': 'message_count',
-                'table': 'chats',
-                'chat_id_col': 'id',
-                'count_col': 'lastMessagePreview_id'
             },
-            'chat_table': {
-                'name': 'chat_messages',
-                'chat_id_col': 'chatId',
-                'username_col': 'from_user_displayName',
-                'text_col': 'body_content',
-                'time_col': 'createdDateTime',
-            }
+            'tables': [
+                {
+                    'polling': {
+                        'type': 'message_count',
```

Comment:
what is the purpose of  'type': 'message_count' here if we have the same setting in the line 146? 
we can't use 'realtime' here

Reply:
@ea-rus That was a mistake; it is not required.